proposals/dao/tip_118.ts

```diff
@@ -273,8 +273,8 @@ const validate: ValidateUpgradeFunc = async (addresses, oldContracts, contracts,
   console.log('----------------------------------------------------');
 
   // PCV Equity change should be neutral for this proposal
-  expect(Number(eqDiff) / 1e18).to.be.at.least(-10000);
-  expect(Number(eqDiff) / 1e18).to.be.at.most(+10000);
+  // expect(Number(eqDiff) / 1e18).to.be.at.least(-10000);
+  // expect(Number(eqDiff) / 1e18).to.be.at.most(+10000);
```

Comment:
pushed the fix in #922 you can update your base branch ^^

Reply:
Nice, done